Probesys/bileto
# misc: Rename the phpunit config file

Changes proposed in this pull request:

- Rename the phpunit config file in `.phpunit.xml.dist`

How to test the feature manually:

1. Run the tests and verify they are still running

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
